README.md

```diff
@@ -256,6 +261,13 @@ reqData = new HashMap<String,String>();
 reqData.put("limit", "10");
 
 DataApi dataApi = new DataApi("https://data.learnosity.com/latest/itembank/items", sec, consumerSecret, reqData, "get");
+
+// you can also configure the Data API with a requestConfig as above
+RequestConfig requestConfig = RequestConfig.custom().setSocketTimeout(1000).build();
+
+dataApi.modifyRequest(requestConfig);
```

Comment:
Or we could rename the method to be `modifyRequestConfig` and make it really declarative on what we are doing.